lib/jekyll-seo-tag/drop.rb

```diff
@@ -98,7 +98,7 @@ def date_modified
           date = if page["seo"] && page["seo"]["date_modified"]
                    page["seo"]["date_modified"]
                  else
-                   page["last_modified_at"] || page["date"]
+                   page["last_modified_at"].to_liquid || page["date"]
```

Comment:
Actually, this needs to be 

```ruby
elsif page["last_modified_at"]
  page["last_modified_at"].to_liquid
else
  page["date"]
```

to handle the case where `last_modified_at` is `nil` (where calling `to_liquid` on `nil` would throw an error)